ctrd/watch.go

```diff
@@ -128,7 +127,7 @@ func (w *watch) get(id string) (*containerPack, error) {
 
 	pack, ok := w.containers[id]
 	if !ok {
-		return pack, errtypes.ErrNotfound
+		return pack, fmt.Errorf("failed to find container %s in metadata", id)
```

Comment:
I think we still need to use `errtypes.ErrNotfound`, since `errtypes.ErrNotfound` is a specific error containing a useful error message `not found` for the uppper layer.
We usually use the package `errtypes` to keep align with the upper calling system. When we could take advantages of this package, we should do that. Otherwise we are weakening usage of it. WDYT?

Back to the change here, I think we could use `errors.Warpf(errtypes.ErrNotfound, "container %s in metadata", id)`. How about that?

Reply:
Agree with @allencloud

Reply:
I think the format look weird in your advise, since I think this way at first, and give up. @allencloud @fuweid . I think errtypes.ErrNotfound should be used in single but not the warp way, or you can give me a example to fix me, thanks.

Reply:
If with no `not found` in the returned message, the status code will change from 404 to 500.